services/workflows-service/src/customer/customer.controller.internal.ts

```diff
@@ -15,7 +15,7 @@ export class CustomerControllerInternal {
   @common.Get()
   @swagger.ApiOkResponse({ type: [CustomerModel] })
   @swagger.ApiForbiddenResponse()
-  async find(@ProjectIds() projectIds: TProjectIds): Promise<Customer | null> {
+  async find(@CurrentProject() projectIds: TProjectId): Promise<Customer | null> {
```

Comment:
wrong decorator.
should be @projectIds and you are retrieving projecTIds
or extract single project id ( below )